main/src/ca/uwaterloo/flix/tools/pkg/SemVer.scala

```diff
@@ -25,6 +25,15 @@ object SemVer {
       .orElseBy(_.minor)
       .orElseBy(_.patch)
 
+  def ofString(input: String): Option[SemVer] = {
```

Comment:
It is used in the `getFlixVersion` function. It should ideally also be used in the `perseSemVer` function in `Github.scala` to avoid code duplication.